po/ubuntu-system-settings.pot

```diff
@@ -3048,833 +3062,854 @@ msgstr ""
+#: ../obj-arm-linux-gnueabihf/po/settings.js:76
 msgid "handsfree"
 msgstr ""
 
 #. TRANSLATORS: This is a keyword or name for the bluetooth plugin which is used while searching
 #. TRANSLATORS: This is a keyword or name for the wifi plugin which is used while searching
-#: ../build/po/settings.js:80 ../build/po/settings.js:194
+#: ../obj-arm-linux-gnueabihf/po/settings.js:78
+#: ../obj-arm-linux-gnueabihf/po/settings.js:278
 msgid "wireless"
 msgstr ""
 
 #. TRANSLATORS: This is a keyword or name for the bluetooth plugin which is used while searching
-#. TRANSLATORS: This is a keyword or name for the wifi plugin which is used while searching
 #. TRANSLATORS: This is a keyword or name for the vpn plugin which is used while searching
-#: ../build/po/settings.js:82 ../build/po/settings.js:200
-#: ../build/po/settings.js:404
+#. TRANSLATORS: This is a keyword or name for the wifi plugin which is used while searching
+#: ../obj-arm-linux-gnueabihf/po/settings.js:80
+#: ../obj-arm-linux-gnueabihf/po/settings.js:114
+#: ../obj-arm-linux-gnueabihf/po/settings.js:284
 msgid "connect"
 msgstr ""
 
 #. TRANSLATORS: This is a keyword or name for the bluetooth plugin which is used while searching
-#. TRANSLATORS: This is a keyword or name for the wifi plugin which is used while searching
 #. TRANSLATORS: This is a keyword or name for the vpn plugin which is used while searching
-#: ../build/po/settings.js:84 ../build/po/settings.js:202
-#: ../build/po/settings.js:406
+#. TRANSLATORS: This is a keyword or name for the wifi plugin which is used while searching
+#: ../obj-arm-linux-gnueabihf/po/settings.js:82
+#: ../obj-arm-linux-gnueabihf/po/settings.js:116
+#: ../obj-arm-linux-gnueabihf/po/settings.js:286
 msgid "disconnect"
 msgstr ""
 
 #. TRANSLATORS: This is a keyword or name for the bluetooth plugin which is used while searching
-#: ../build/po/settings.js:86
+#: ../obj-arm-linux-gnueabihf/po/settings.js:84
 msgid "stereo"
 msgstr ""
 
-#. TRANSLATORS: This is a keyword or name for the mouse plugin which is used while searching
-#: ../build/po/settings.js:90
-msgid "mouse"
-msgstr ""
-
-#. TRANSLATORS: This is a keyword or name for the mouse plugin which is used while searching
-#: ../build/po/settings.js:92
-msgid "touchpad"
+#. TRANSLATORS: This is a keyword or name for the hotspot plugin which is used while searching
+#. TRANSLATORS: This is a keyword or name for the vpn plugin which is used while searching
+#. TRANSLATORS: This is a keyword or name for the flight-mode plugin which is used while searching
+#. TRANSLATORS: This is a keyword or name for the wifi plugin which is used while searching
+#. TRANSLATORS: This is a keyword or name for the cellular plugin which is used while searching
+#: ../obj-arm-linux-gnueabihf/po/settings.js:88
+#: ../obj-arm-linux-gnueabihf/po/settings.js:112
+#: ../obj-arm-linux-gnueabihf/po/settings.js:138
+#: ../obj-arm-linux-gnueabihf/po/settings.js:276
+#: ../obj-arm-linux-gnueabihf/po/settings.js:368
```

Comment:
Any idea why it's putting files from the build directory into the translations file? It appears that it's happened before, do we fix it now?